net-dns/dnsdist/files/dnsdist.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=dnsdist
+Wants=network-online.target
+After=network-online.target
```

Comment:
That is how upstream does it.

I wanted to have systemd support covered, too. However, I stick to OpenRC and honestly don't know.

To run dnsdist you'll need network (whatever that is), at least an interface to bind to.